js/components/ConsortReport.vue

```diff
@@ -9,21 +9,24 @@
       </ul>
     </div>
 
```

Comment:
I added a spinner to the loading indicator and overrode the styles to remove the red border from REDCap's stylesheet.